README.md

```diff
@@ -130,6 +131,3 @@ pack.pipe(newTarballStream)
 
 [See tar-fs for a performance comparison with node-tar](https://github.com/mafintosh/tar-fs/blob/master/README.md#performance)
 
-# License
-
-MIT
```

Comment:
Why have you removed this license field? Why don't have both this and the badge?